.gitignore

```diff
@@ -5,6 +5,9 @@ frontend/static/symbols/mtb/*
 frontend/static/symbols/riding/*
 frontend/static/symbols/skating/*
 frontend/static/symbols/slopes/*
+frontend/static/.webassets-cache/*
+frontend/static/packed/*
+data.osm.pbf
```

Comment:
The first two are okay but the third is docker specific and does not belong in this file. I would suggest to move it into the $HOME/.gitignore file of the user inside the docker instance.